src/libraries/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -290,29 +291,24 @@ public async Task SendAsync_ServerDoesntExist_Throws()
         [Fact]
         public void TestMailDelivery()
         {
-            SmtpServer server = new SmtpServer();
-            SmtpClient client = new SmtpClient("localhost", server.EndPoint.Port);
-            client.Credentials = new NetworkCredential("user", "password");
+            const string Username = "Foo";
+            const string Password = "Bar";
```

Comment:
This construct might trigger our automatic static security analysis tool. It checks for hard-coded passwords in source code.

While this is just test code, we usually do something like this to generate the password:

```c#
readonly string Password = Guid.NewGuid();
```